### PR TITLE
changes on publications

### PR DESCRIPTION
**Pull request by knil-maloon**
New variables. Changes on logic and table search.

### CODE REVIEW

File: crontasks/cron_publications.php [hunk 1]
```diff
@@ -71,12 +74,12 @@
         #Regional Content
         foreach ($extra_outputs as $output) {
             $records++;
-            $type = "<span class='badge badge-pill badge-draft'>".$settings['pubtext4']."</span>";
+            $type = "<span class='badge badge-pill badge-draft'>".$pubtext3."</span>";
             if ($output['producedby_region'] == 2) {
-                $type = "<span class='badge badge-pill badge-draft'>".$settings['pubtext3']."</span>";
+                $type = "<span class='badge badge-pill badge-draft'>".$pubtext5."</span>";
```
Comment (kcmcg) [inline crontasks/cron_publications.php:79]: Was this wrong before? I'm confused about why it's changing from pubtest3 to pubtest5. Same above for pubtext4 to pubtext3 and below for pubtext3 to pubtext4

Comment (knil-maloon) [reply crontasks/cron_publications.php:79]: No, they just wanted it differently now, heh.

**Review by kcmcg — APPROVED** [review]
Ok, just wanted to make sure the change was intentional.

File: crontasks/cron_publications.php (hunk 1 above)
Comment (knil-maloon) [reply crontasks/cron_publications.php:79]: I completely understand, hehe. Thanks!

**Pull request merged**